software_training/src/trainingNode.cpp

```diff
@@ -0,0 +1,91 @@
+#include "Turtle.h"
+
+#include "ros/ros.h"
+
+#include "turtlesim/Kill.h"
+#include "turtlesim/Spawn.h"
+#include "turtlesim/Pose.h"
+
+class Turtle {
+    public:
+        // ctor
+        Turtle(ros::NodeHandle &n, float x, float y, std::string name);
+
+        // kill `name` turtle
+        static bool kill(ros::NodeHandle &n, std::string name);
```

Comment:
making it static and taking in the name defeats the purpose of the class, make it a member function